index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
The current wording is not just "hand-wavy" it's incorrect. The intent when this was designed was that a handler matches by default UNLESS something provided in the `data` part of the request excludes it.

The current wording suggests that a payment handler ONLY matches if the appropriate `data` is provided.

Example: If the handler supports `basic-card` and supported networks of `visa` and `mastercard` the current text suggests that the payment request MUST list `visa` or `mastercard` under `supportedNetworks` in the `data`  of the request or it will not match.

The intent is that a payment request with a method of `basic-card` and an empty `data` should match. i.e Empty implies, "no filter applied". 

We've used the terminology "filters" for requests and "capabilities" for handlers for some time so not sure I understand this, but whatever...

> I can't accept a PR with the word "filter".